### PR TITLE
Increase test coverage of english language generator

### LINKED ISSUE #601: Try to push test coverage on english_language_generator to close to 100%

### PR DESCRIPTION
**Pull request by LichMaster98**
Closes #601

### CODE REVIEW

**Comment by gabbard** [comment]
@LichMaster98 : what is the state of this PR?

**Comment by LichMaster98** [comment]
@gabbard It's not 100% in test coverage but most of the remaining tests are for odd cases. It's fine to review this and merge it in and I can work on the last 10% of coverage when time allows.

**Comment by gabbard** [comment]
@LichMaster98 : Can you let me know when you've resolved conflicts with `master` and the I will review?

File: adam/language_specific/english/english_language_generator.py [hunk 1]
```diff
@@ -560,7 +555,9 @@ def _translate_verb_argument(
                             preposition, filler_noun, role=CASE_SPATIAL
                         )
                 return (syntactic_role, filler_noun)
-            elif isinstance(filler, Region):
+            # As our filler is guaranteed to either be a Region or an Object at this point
```
Comment (gabbard) [inline adam/language_specific/english/english_language_generator.py:558]: The value of being more defensive is that if this assumption ever changes in the future, we will get an exception alerting us this code needs to be updated.

**Pull request merged**